### DISCUSSION TITLE
Liked Music and Favorites

### DISCUSSION THREAD

**Discussion by Bwangster12** [maxrave-dev/SimpMusic#215] [Ideas]
Anyway to integrate favoriting a song with the Liked Songs synced from YouTube?  It doesnt favorite to the same list.

**Comment by chamudha** [comment]
Can add a way to disable this please  ??![IMG_20240129_195329.png](https://github.com/maxrave-dev/SimpMusic/assets/134184550/5d9fb46f-1078-4a4d-b060-f973f8a4a1cb)
